src/orchestrator/src/lib.rs

```diff
@@ -385,3 +390,37 @@ impl Serialize for CpuLimit {
         <f64 as Serialize>::serialize(&(self.millicpus as f64 / 1000.0), serializer)
     }
 }
+
+/// Describes a limit on disk usage.
+#[derive(Copy, Clone, Debug, PartialOrd, Eq, Ord, PartialEq)]
+pub struct DiskLimit(pub ByteSize);
+
+impl DiskLimit {
+    pub const MAX: Self = Self(ByteSize(u64::MAX));
+    pub const ONE_GIB: Self = Self(ByteSize::gib(1));
```

Comment:
Can we call this `ARBITRARY` or something to make clear that the specific value doesn't matter? 

```suggestion
    pub const ARBITRARY: Self = Self(ByteSize::gib(1));
```